files_management/task_04.py

```diff
@@ -0,0 +1,30 @@
+import csv
+
+filename = "students.csv"
+
+students_data = [
+    ["Name", "Age", "Grade"],
+    ["Alice", 25, "A"],
+    ["Bob", 22, "B"],
+    ["Charlie", 24, "A"]
+]
+
+with open(filename, "w", newline="", encoding="utf-8") as file:
+    writer = csv.writer(file)
+    writer.writerows(students_data)
+
+print(f"File '{filename}' created.")
+
+ages = []
+print("\nFile content:")
+with open(filename, "r", encoding="utf-8") as file:
+    reader = csv.reader(file)
+    next(reader)
+    for row in reader:
```

Comment:
In this case it would be better to use slices

```suggestion
    for row in reader[1:]:
```